src/components/questionnaire/questionnaire-table.tsx

```diff
@@ -18,59 +18,97 @@ import strings from "src/localization/strings";
 import UserRoleUtils from "src/utils/user-role-utils";
 import { DataGrid, type GridRenderCellParams } from "@mui/x-data-grid";
 import { useLambdasApi } from "src/hooks/use-api";
+import { useSetAtom } from "jotai";
+import { errorAtom } from "src/atoms/error";
 
 /**
  * Questionnaire Table Component
  * @returns Questionnaires from DynamoDB rendered in a x-data-grid table
  */
 const QuestionnaireTable = () => {
   const adminMode = UserRoleUtils.adminMode();
-  const { questionnaireApi } = useLambdasApi();
+  const { questionnairesApi } = useLambdasApi();
   const [questionnaires, setQuestionnaires] = useState<Questionnaire[]>([]);
-  const [loading, setLoading] = useState(true);
+  const [loading, setLoading] = useState(false);
   const [dialogOpen, setDialogOpen] = useState(false);
   const [deleteId, setDeleteId] = useState<string | null>(null);
   const [deleteTitle, setDeleteTitle] = useState<string | null>(null);
+  const setError = useSetAtom(errorAtom);
 
   useEffect(() => {
     const fetchQuestionnaires = async () => {
       setLoading(true);
       try {
-        const questionnaires = await questionnaireApi.listQuestionnaires();
+        const questionnaires = await questionnairesApi.listQuestionnaires();
         setQuestionnaires(questionnaires);
-      } finally {
-        setLoading(false);
+      } catch (error) {
+        setError(`${strings.error.questionnaireLoadFailed}, ${error}`);
       }
+      setLoading(false);
     };
     fetchQuestionnaires();
   }, []);
 
+  /**
+   * Function to open the dialog for deleting the questionnaire
```

Comment:
```suggestion
   * Function to open the dialog for deleting the questionnaire
   *
```

Reply:
There should be a space between TSDoc description and the @param. Update when you notice it please- it's wrong in loads of places